src/main/java/com/naitei/group3/movie_ticket_booking_system/controller/api/AuthApiController.java

```diff
@@ -0,0 +1,34 @@
+package com.naitei.group3.movie_ticket_booking_system.controller.api;
+
+import com.naitei.group3.movie_ticket_booking_system.dto.request.RegisterRequestDTO;
+import com.naitei.group3.movie_ticket_booking_system.enums.RoleType;
+import jakarta.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import com.naitei.group3.movie_ticket_booking_system.entity.User;
+import com.naitei.group3.movie_ticket_booking_system.service.impl.UserService;
+
+@RestController
+public class AuthApiController {
+  private final UserService userService;
+
+  public AuthApiController(UserService userService) {
+    this.userService = userService;
+  }
+
+
+  @PostMapping("/registerApi")
```

Comment:
hay đấy nhưng chưa đẹp 
```suggestion
  @PostMapping("api/v1/register")
```
`api/v1/` lafg giá trị default bắt đầu của các action  client